logstash-core/src/main/java/org/logstash/ext/JrubyWrappedSynchronousQueueExt.java

```diff
@@ -0,0 +1,47 @@
+package org.logstash.ext;
+
+import org.jruby.Ruby;
+import org.jruby.RubyClass;
+import org.jruby.RubyNumeric;
+import org.jruby.RubyObject;
+import org.jruby.anno.JRubyClass;
+import org.jruby.anno.JRubyMethod;
+import org.jruby.runtime.ThreadContext;
+import org.jruby.runtime.builtin.IRubyObject;
+
+import java.util.concurrent.ArrayBlockingQueue;
+import java.util.concurrent.BlockingQueue;
+
+@JRubyClass(name = "WrappedSynchronousQueue")
+public class JrubyWrappedSynchronousQueueExt extends RubyObject {
```

Comment:
I know that these are following the existing `Jruby...Ext` pattern that already exists in the codebase, but I'm really not a fan of that naming convention -- it doesn't add much clarity (e.g., when I come across it, the `Jruby` bit _kindof_ hints that it's a `RubyObject` and has ruby-ish things about it, but otherwise doesn't tell me much).

Reply:
I thought that naming convention was reasonable since these are JRuby extension classes. Do you have a suggestion for a better naming convention?

As a related aside, I expect there will be a growing number of JRuby extension classes in the earlier phases of the processing pipeline's move to Java. Once the pipeline is more Java than Ruby and the interface point between Ruby and Java can be minimized, I expect a number of these classes will be converted to pure Java to eliminate the unnecessary dependency on JRuby.